plugins/base/src/main/kotlin/translators/documentables/PageContentBuilder.kt

```diff
@@ -229,6 +229,10 @@ open class PageContentBuilder(
             block: DocumentableContentBuilder.() -> Unit
         ): ContentGroup = contentFor(dri, platformData, kind, styles, extra, block)
 
+        fun breakLine(platformData: Set<PlatformData>) {
```

Comment:
```suggestion
        fun breakLine(platformData: Set<PlatformData> = mainPlatformData) {
```
and the call looks like this:
`breakLine()`